docs/component-spec/web-services.md

```diff
@@ -94,17 +95,43 @@ Web services *must* implement the following endpoints, for each version of the a
 <table class="o-techdocs-table">
 <tr><td><code>/</code></td><td>Description of the service and instructions for use, designed for human consumption.  This <em>should</em> be HTML, and <em>may</em> choose to use the standard Origami documentation stylesheet.</td></tr>
 <tr><td><code>/__health</code></td><td>Health status JSON data conforming to the <a href="https://docs.google.com/a/ft.com/document/d/18hefJjImF5IFp9WvPAm9Iq5_GmWzI9ahlKSzShpQl1s/edit">FT Health check standard</a>.  Note that the health check standard currently requires the alerts to be output in a 'human readable' form, and that may require implementing additional endpoints (or reformatting at the edge)</td></tr>
-<tr><td><code>/__metrics</code></td><td>A JSON document listing current metrics to allow automated monitoring, in the <a href='{{site.baseurl}}/docs/syntax/metrics'>metrics</a> format</td></tr>
 <tr><td><code>/__about</code></td><td><em>(root only)</em> A JSON document linking to all available versions of the service, in the <a href='{{site.baseurl}}/docs/syntax/web-service-index'>web service index format</a><br/><em>(version endpoints only)</em> A JSON document in the <a href='{{site.baseurl}}/docs/syntax/web-service-description'>web service description format</a></td></tr>
 </table>
 
 If a web service has two versions, `v1` and `v2`, there *must* be three of each of the above.  Using the `/health` endpoint as an example, the complete paths `/__health`, `/v1/__health` and `/v2/__health` *must* be recognised and served by the web service, and *may* return the same content.  The `__about` data *must* also be available from three URLs, but will follow the [web service description format]({{site.baseurl}}/docs/syntax/web-service-description) format for those that are version prefixed, and the [web service index]({{site.baseurl}}/docs/syntax/web-service-index) format for the one that isn't.
 
-
 ### De-duplication of output
 
 Web service components *should* not offer any de-duplication of content.  If a product developer wants to draw from multiple Origami sources, and de-dupe where the same individual content item may appear from more than one of those sources, that's not a problem that Origami will solve for them, but could be solved at the product level by consuming data rather than markup.
 
+## Metrics
+
+Web service components *should* emit metrics over TCP or UDP using [any protocol supported by Carbon](http://graphite.readthedocs.org/en/latest/feeding-carbon.html), to the FT Graphite service.  All metrics *must* conform to the [metrics naming conventions]({{site.baseurl}}/docs/syntax/metrics).
```

Comment:
I'd say TCP only using the line protocol.  This is primarily because we ([or I](https://github.com/samgiles/graphiteauthd)) need to maintain a 'secure gateway' proxy that understands the incoming protocol.  If we were to advocate UDP from the start we'd also have to implement Datagram Transport Layer Security as well as TLS.  I think for simplicities sake we should stick to TCP.  An additional Layer 7 HTTP endpoint would be trivial.